ckanext/cfpb_extrafields/templates/package/snippets/package_basic_fields.html

```diff
@@ -7,6 +7,25 @@
 {% endblock %}
 
 {% block package_basic_fields_description %}
+
+<div class="core-field">
+  {{ form.markdown('notes', id='field-notes', label=_('description'), placeholder=_('eg. Some useful notes about the data'), value=data.notes, error=errors.notes, is_required=true) }}
+</div>
+{% endblock %}
+
+{% block package_basic_fields_tags %} 
+<div class="core-field">
+{% set tag_attrs = {'data-module': 'autocomplete', 'data-module-tags': '', 'data-module-source': '/api/2/util/tag/autocomplete?incomplete=?'} %}
+{{ form.input('tag_string', id='field-tags', label=_('subject matter'), placeholder=_('eg. economy, mental health, government'), value=data.tag_string, error=errors.tags, classes=['control-full'], attrs=tag_attrs) }}
+</div>
+{% endblock %}
+
+{% block package_basic_fields_org %}
+<div class="core-field">
+  {{ super() }}
+</div>
+{% endblock %}
+{% block package_metadata_fields_visibility %}
 {#----------------------------------------#}
```

Comment:
I actually moved the code between these markers down instead of moving that big block up. Github's diff algorithm just incorrectly interpreted my change.